tests/ResourceSyncingUsingPolymorphicTest.php

```diff
@@ -73,9 +73,7 @@
     $centralUser->tenants()->attach('t1');
 
     // Assert `tenants` are accessible
-    expect($centralUser->tenants->first())
-        ->not()->toBeNull()
-        ->id->toBe('t1');
+    expect($centralUser->tenants->first()->id)->toBe('t1');
```

Comment:
One more change: maybe using `->tenants->pluck('id')` and `toBe(['t1'])` would also assert that there's only one tenant attached.